src/core/tests.py

```diff
@@ -0,0 +1,29 @@
+from django.test import TestCase, Client
+
+class DownstreamCaching(TestCase):
+    def setUp(self):
+        self.c = Client()
+        self.url = '/' # we could hit more urls but it's applied application-wide
+
+    def tearDown(self):
+        pass
+
+    def test_cache_headers_in_response(self):
+        expected_headers = [
+            'vary',
```

Comment:
Better to test the value (`Vary: bananas` would pass?).

Reply:
... could you explain please? Observer doesn't respond with 'bananas' anywhere. Is this a Kong thing? the value of `vary` will be `Accept`...

Reply:
I meant this only tests for the presence of the headers, not that they have meaningful values.